content/outcomes/application-development/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Build Modern Applications
```

Comment:
Recommend changing this to _Modern Application Development_ as the other outcome  titles are all nouns, not verbs.